UnleashClient/features/Feature.py

```diff
@@ -75,20 +82,20 @@ def _count_variant(self, variant_name: str) -> None:
         """
         self.variant_counts[variant_name] = self.variant_counts.get(variant_name, 0) + 1
 
-    def is_enabled(self, context: dict = None) -> bool:
+    def is_enabled(self, context: dict = None, skip_stats: bool = False) -> bool:
```

Comment:
Is there a more elegant way to do it? Without passing `skip_stats` around.

Reply:
Can't think of a way without rearchitecting this a little. It should be possible to move the count metrics methods to public ones and only call those in the is_enabled in the UnleashClient but... I'm not sure that's worth it. This is all internal at the moment and when Yggdrasil arrives here this will go away anyway.

I'm okay with this as it stands. I think we can do better but I also don't think it's worth it to do so. I think this is a reasonable tradeoff